tldw_Server_API/app/core/MCP_unified/catalog_loader.py

```
@@ -0,0 +1,113 @@
+    """
+    _CATALOG_CACHE.clear()
+
+    file_path = Path(path)
+    if not file_path.is_file():
+        logger.warning("MCP catalog file does not exist: {}", file_path)
+        return _CATALOG_CACHE
+
+    try:
+        raw = file_path.read_text(encoding="utf-8")
+        data = yaml.safe_load(raw)
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)